arrow.opam

```diff
@@ -32,15 +25,12 @@ depends: [
   "core_kernel"
   "pyml"
```

Comment:
pyml is only useful for the tests so I would rather not having it as a main dependency. Dependencies can have a `with-test` flag though I'm not sure it's actually for this use case.